src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -161,6 +169,7 @@ class UniversalNavbar extends React.Component {
 UniversalNavbar.propTypes = {
   hideMobileCta: PropTypes.bool.isRequired,
   trackCtaClick: PropTypes.func.isRequired,
+  link: PropTypes.node,
```

Comment:
Can we call this something more descriptive? I would suggest `LinkComponent`, and capitalize it to suggest it's a React component. You can then just call `<LinkComponent>` directly without the extra step above (PropsLink = link).